app/src/main/res/layout/question_player_content_item.xml

```diff
@@ -0,0 +1,30 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="htmlContent"
+      type="CharSequence" />
+  </data>
+
+  <FrameLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="28dp"
+    android:layout_marginTop="36dp"
+    android:layout_marginEnd="28dp"
+    android:layout_marginBottom="28dp">
```

Comment:
I think margin bottom is 36dp in the mock.

Reply:
This will be added to divider_margin_top i.e. 8 dp in the interactions and response.

Reply:
Got it. thanks.